src/documents/migrations/0023_document_searchable_content.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 2.0.13 on 2019-05-30 14:50
+import unicodedata
+
+from django.db import migrations, models
+
+from paperless.utils import slugify as slugifyOCR
```

Comment:
While it can be tempting to import stuff from modules into a migration to keep your code DRY, this will later bite us in the ass should we decide to rename/remove this function.  It will break all migrations for all time as a result.

Instead, if you have logic you wish to make available in a migration, copy it verbatim into the migration.